src/.editorconfig

```diff
@@ -96,4 +92,182 @@ end_of_line = crlf
 dotnet_diagnostic.IDE0065.severity = none
 
 # IDE0009: Add this or Me qualification
-dotnet_diagnostic.IDE0009.severity = none
\ No newline at end of file
+dotnet_diagnostic.IDE0009.severity = none
+
+# IDE-based code analysis rules
+# IDE0005: Remove unnecessary import
+dotnet_diagnostic.IDE0005.severity = suggestion
+
+# IDE0008: Use explicit type instead of 'var'
+dotnet_diagnostic.IDE0008.severity = suggestion
```

Comment:
I would defer to @crutkas for this one. We can surely tweak, but I wonder about those who have all info's filtered out and use the suggestions to improve incrementally. At the same time, I can understand it in the situations where squiggles are everywhere.

Reply:
`silent` will still allow changes, for example via light-bulb actions.

IMO, the following should be `silent` and not `suggestion`. Making them `suggestion` indicates that performing the suggested refactoring will improve the code, when in fact the opposite is likely true:

# IDE0008: Use explicit type instead of 'var'
# IDE0061: Use block body for methods
# IDE0160: Use block-scoped namespace

Reply:
That makes sense! I'm currently in the car for the next 3.5 hours, but if you want to make those changes that would be appreciated.

Reply:
While trying to fix this, I noticed that almost all of the comments are incorrect, i.e. the style codes don't seem to correspond to the actual ones listed [here](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/). Could you first address this please?

Reply:
Great catch. I thought I fixed them all from reorganizing the comments 😅. I'll fix these in a bit since I just stopped to grab a quick bite to eat

Reply:
There we go. Fixed. I know exactly what happened and I'm glad you caught it. That's what I get for asking Copilot to sort 🙃